Tracker/Onboarding/OnboardingPage.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+
+final class OnboardingPage: UIViewController {
+    private(set) var pageIndex: Int?
+
+    private let images = [
+        UIImage(named: "Onboarding_1"),
+        UIImage(named: "Onboarding_2"),
+    ]
+
+    private let titles: [String] = [
+        "Отслеживайте только то, что хотите",
+        "Даже если это  не литры воды и йога",
+    ]
```

Comment:
Было бы классно обернуть инфу о странице(текст и картинку) в enum. [Сделать эту инфу вычислимыми свойствами](https://www.avanderlee.com/swift/computed-property/). Так код будет гибче.
```swift
enum PageModel {
  case somePage
  case anotherPage

  var image : UIImage { ... }
  var text : String { ... }
}

...

let secondPageVC = OnboardingPage(pageModel : .anotherPage)
```

Reply:
Если добавится третья страница то она будет называться `anotherAnotherPage`, а следующая `anotherAnotherAnotherPage` и так далее.
В случае с массивом страницу можно легко добавить и удалить, и если нужно то так же передавать в инициализатор.
Например, объединить в один массив можно было бы так:
```suggestion
    private let pages: [PageModel] = [
        .init(image: UIImage(named: "Onboarding_1"), title: "Отслеживайте только то, что хотите"),
        .init(image: UIImage(named: "Onboarding_2"), title: "Даже если это  не литры воды и йога"),
    ]
```
Поэтому не очень понимаю в чем гибкость enum в данном случае.

Reply:
> Кажется, что это более гибкое решение

Такой вариант тоже неплох🤷

Reply:
> она будет называться `anotherAnotherPage`, а следующая `anotherAnotherAnotherPage` и так далее

`anotherPage`  - название для примера :)  Очевидно, что названия нужно давать более осмысленные. Например:
```swift
enum PageModel {
  case aboutTracking
  case aboutWaterAndYoga 
  ...
}
```